### PR TITLE
docs: Add instructions for VMware Player on Linux

### PR DESCRIPTION
**Pull request by sbrownjc**
In attempting to get packer working with VMWare Workstation Player on Linux, I had a lot of trouble until I found the information scattered about the internet, and then found this closed PR, whose instructions mirrored exactly what I'd done to get Workstation Player 16 running with Packer: https://github.com/hashicorp/packer/pull/3451

This closed issue https://github.com/hashicorp/packer/issues/1102 references this (lost?) commit: https://github.com/hashicorp/packer/commit/c5ec92c88bb154bd317aec5b81ba6cf9e4dff9c4 which would have saved me much time if I could have seen it, so I'm making this PR to bring the information from https://github.com/hashicorp/packer/pull/3451 in and up-to-date.

### CODE REVIEW

**Comment by nywilken** [comment]
👋 I still think that this [comment](https://github.com/hashicorp/packer/pull/3451#issuecomment-366091480) is still very relevant. That said, @tenthirtyam do you have some thoughts on how we should be capture this. 

Is this something that you have documented within your vmware-samples repo?

**Comment by tenthirtyam** [comment]
Hey, @nywilken -

I don't have anything in [that](https://github.com/vmware-samples/packer-examples-for-vsphere) project that uses the `vmware-iso` builder. 

I do use the `vmware-iso` builder for VMware Fusion and VMware Workstation in the https://github.com/vmware/photon-packer-templates project to create Photon images for Vagrant. However, VMware Player is not included/tested.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.

**Comment by tenthirtyam** [comment]
I've added some comments to this PR for context, clarity, and correctness.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.

File: docs/builders/index.mdx
Comment (tenthirtyam) [inline docs/builders/index.mdx:30]: "VMware Workstation Player on Linux"

File: docs/builders/index.mdx
Comment (tenthirtyam) [inline docs/builders/index.mdx:32]: "VMware Workstation Player"

File: docs/builders/index.mdx
Comment (tenthirtyam) [inline docs/builders/index.mdx:39]: I would remove the versioning. Ideally the plugin should support only those versions supported by VMware per the lifecycle matrix. See project README.

**Review by tenthirtyam — CHANGES_REQUESTED** [review]
@nywilken @sbrownjc, I've recommended some changes before merge.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.

**Comment by sbrownjc** [comment]
@tenthirtyam Thanks for the review comments, I've updated the docs accordingly.

**Comment by tenthirtyam** [comment]
Thanks! I'll review the updates tomorrow for Wilken and provide and update.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.

**Review by tenthirtyam — APPROVED** [review]
LGTM! 🚀 

cc @nywilken

**Comment by nywilken** [comment]
@tenthirtyam thanks for the thoughts and review on this PR, and @sbrownjc thanks for updates. The changes look good me me. We just need to have you sign the CLA so that I can merge these changes in. 

If you have already accepted the CLA please make sure that the email used in your commits match the email you signed the CLA with and that the signing email is your primary GitHub email. The CLA bot only checks against the user's primary GitHub email.

**Comment by tenthirtyam** [comment]
Hi @sbrownjc -  could you resolve the issue with signing the HashiCorp CLA? This will help expedite your valuable updates.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.

**Comment by tenthirtyam** [comment]
Hi @sbrownjc - could you resolve the issue with signing the HashiCorp CLA? This will help expedite your valuable updates.

Ryan Johnson
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.

**Comment by sbrownjc** [comment]
Hi @tenthirtyam, I'm sorry for the delay, I was waiting on a response from legal since I posted using my work GH account.

If possible, I would like to submit this work outside of a CLA, following section 7 of the CLA:
> Should You wish to submit work that is not Your original creation, You may submit it to HashiCorp separately from any Contribution, identifying the complete details of its source and of any license or other restriction (including, but not limited to, related patents, trademarks, and license agreements) of which you are personally aware, and conspicuously marking the work as "Submitted on behalf of a third-party: [named here]".

This work is being Submitted on behalf of a third-party: [Anand Kumria](https://github.com/akumria)

**Comment by tenthirtyam** [comment]
cc @nywilken for submission state review.

**Comment by nywilken** [comment]
@sbrownjc @tenthirtyam apologies for not getting back much sooner. In speaking with our legal team the author of the change must still sign the CLA in order to make the contribution on behalf of another contributor under section 7. Unfortunately that means I will not be able to merge in these changes unless the CLA is signed.

I'm going to close this PR given the time I kept everyone waiting. If you wish to reopen with your personal GH and sign the CLA I would be glad to merge it right away. 

Thank you @tenthirtyam for your help here.

**Pull request closed without merging**
